gemrb/core/System/String.cpp

```diff
@@ -100,6 +105,47 @@ static String* StringFromEncodedData(const ieByte* string, const EncodingStruct&
 	return dbString;
 }
 
+// returns new string converted to given encoding
+// in case iconv is not available, or there is encoding error, copy of original
+// string is returned
+const char* ConvertCharEncoding(IGNORE_UNUSED const char* string,
+        IGNORE_UNUSED const char* from, IGNORE_UNUSED const char* to) {
+#if HAVE_ICONV
```

Comment:
I think it would be cleaner to do this:

```
#if HAVE_ICONV
const char* ConvertCharEncoding(const char* string, const char* from, const char* to)
{
...
}
#else 
const char* ConvertCharEncoding(const char*,  const char*, const char*)
{
...
}
#endif
```

that also fixes the unused parameter warnings without allowing them for the iconv case